resources/idlharness.js

```diff
@@ -2067,6 +2067,50 @@ IdlInterface.prototype.add_iterable_members = function(member)
           [{ name: "callback", idlType: {idlType: "function"}},
            { name: "thisValue", idlType: {idlType: "any"}, optional: true}]}));
 };
+//@}
+IdlInterface.prototype.add_maplike_members = function(member)
+//@{
+{
+    this.members.push(new IdlInterfaceMember(
+        { type: "operation", name: "size", idlType: "maplike", arguments: []}));
+    this.members.push(new IdlInterfaceMember(
+        { type: "operation", name: "entries", idlType: "maplike", arguments []}));
```

Comment:
The problem is a missing : here and elsewhere.

Reply:
I will change it soon. Thank you!